src/ngraph/runtime/cpu/unit_test.manifest

```diff
@@ -28,3 +28,6 @@ random_uniform_all_static_seed_used
 random_uniform_seed_use_dynamic
 random_uniform_all_static_range_dynamic
 random_uniform_dynamic_shapes
+
+# Failing on CPU
+lstm_cell_no_bias_no_peepholes
```

Comment:
Thats weird because default bias and peepholes should be equal to zero.

Reply:
Could you add an unit test with bias and peepholes filled with default values?
If this kind of test passed, that would mean something strange.